components/services/searcher/statefulset/patches/searcher.StatefulSet.yaml

```diff
@@ -1,49 +1,58 @@
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: symbols
+  name: searcher
 spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: searcher
+  serviceName: searcher
   template:
+    metadata:
+      labels:
+        app: searcher
+        deploy: sourcegraph
     spec:
       containers:
-      - env:
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: CACHE_DIR
-          value: /mnt/cache/$(POD_NAME)
-        - name: OTEL_AGENT_HOST
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
-        - name: OTEL_EXPORTER_OTLP_ENDPOINT
-          value: http://$(OTEL_AGENT_HOST):4317
-        name: symbols
-        resources:
-          limits:
-            cpu: "2"
-            memory: 2G
-          requests:
-            cpu: 500m
-            memory: 500M
-        securityContext:
-          allowPrivilegeEscalation: false
-          runAsGroup: 101
-          runAsUser: 100
+        - env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: CACHE_DIR
+              value: /mnt/cache/$(POD_NAME)
+            - name: OTEL_AGENT_HOST
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: http://$(OTEL_AGENT_HOST):4317
+          name: searcher
+          resources:
+            limits:
+              cpu: "2"
+              memory: 2G
+            requests:
+              cpu: 500m
+              memory: 500M
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsGroup: 101
+            runAsUser: 100
       securityContext:
         fsGroup: 101
         fsGroupChangePolicy: OnRootMismatch
         runAsUser: 100
   updateStrategy:
     type: RollingUpdate
   volumeClaimTemplates:
-  - metadata:
-      name: cache-ssd
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      resources:
-        requests:
-          storage: 50G
-      storageClassName: sourcegraph
\ No newline at end of file
+    - metadata:
+        name: cache-ssd
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 50G
```

Comment:
seems like this never requested a specific storage class -- weird but I don't really want to change this now and cause deployment issues, wdyt?

Reply:
It's likely a mistake it never requested a storageClass to begin with 🙁
If we add it now though, K8s won't migrate the data (it won't delete it either), but the PVC would be recreated and be empty which would likely cause issue with our customers.

Reply:
right, that's why I was hesitant to add it 🙈